index.js

```diff
@@ -5,12 +5,14 @@ var path = require('path')
 var serveStatic = require('serve-static')
 var serveStaticFile = require('connect-static-file')
 var compression = require('compression')
+var log4js = require('log4js')
 var app = connect()
 
 const PORT = 9000
 const DIRECTORY = 'public'
 const FILE = 'index.html'
 const HOST = '0.0.0.0'
+const LOGGEROPTIONS = {}
```

Comment:
I think the default value for this should be false. Then we could check if it's true or an object, then attach the logger.